src/Hazelcast.Net.Tests/Clustering/MemberConnectionTests.cs

```diff
@@ -317,7 +317,8 @@ private async ValueTask ServerHandler(ClientRequest<ServerState> request)
                         var authRequest = ClientAuthenticationServerCodec.DecodeRequest(request.Message);
                         var authResponse = ClientAuthenticationServerCodec.EncodeResponse(
                             0, request.State.Address, request.State.MemberId, SerializationService.SerializerVersion,
-                            "4.0", partitionsCount, request.Server.ClusterId, false);
+                            "4.0", partitionsCount, request.Server.ClusterId, false,
+                            Enumerable.Empty<int>().ToList(),Array.Empty<byte>());
```

Comment:
Can't that be `Array.Empty<int>()` here and in the files below?

Reply:
It can be, fixed at 1e035072a014026fe3dd6f8b5a3f8970f4faf5a2